### PR TITLE
feat(OHI IBM MQ): Added branching doc structure

### PR DESCRIPTION
**Pull request by nbaenam**
Added the branching doc structure for this page: [IBM MQ monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration).

This is the ticket: NR-217451

### CODE REVIEW

File: src/install/ibm-mq/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import infrastructureIbmmqDashboard from 'images/infrastructure_screenshot-crop_ibmmq-dashboard.webp'
+
+<Callout title="Public preview">
+  This feature is currently a public preview.
+</Callout>
+
+Our IBM MQ [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations)
+collects and sends dimensional metrics from [IBM MQ](https://www.ibm.com/products/mq).
+
+You can view this [metric data](#metrics) in pre-built <InlinePopover type="dashboards" />, create <InlinePopover type="alerts" /> policies, and create custom queries and charts.
```
Comment (bradleycamacho) [inline src/install/ibm-mq/intro.mdx:15]: I'd remove these popovers. They're nice in a way to introduce users to these NR concepts, but we want as few links out of the doc as possible (so users don't get lost)

File: src/install/ibm-mq/intro.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  title="IBM MQ quickstart dashboard"
+  alt="Image of the dashboard available through the IBM MQ quickstart"
+  src={infrastructureIbmmqDashboard}
+/>
+
+<figcaption>
+  Dashboard installed through the [New Relic IBM MQ Monitor quickstart](https://one.newrelic.com/nr1-core?state=c95ed954-7277-e44d-7fd7-351a77b3ff37).
+</figcaption>
+
+To get data from IBM MQ infrastructure, our IBM MQ integration uses a third-party [Prometheus exporter](https://github.com/ibm-messaging/mq-metric-samples/tree/master/cmd/mq_prometheus), which exposes Prometheus metrics on the port specified in the configuration. The integration collects these metrics, transforms them into entities, filters them, and then sends them to New Relic.
```
Comment (bradleycamacho) [inline src/install/ibm-mq/intro.mdx:27]: Not 100% sure, but "transforms them into entities" doesn't seem right. The specific infra being monitored by this integration install would be the entity.

I'd just cut out the "transforms them into entities"

File: src/install/ibm-mq/intro.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
+<Callout variant="important">
+  The IBM MQ integration doesn't support ARM64 architecture.
+</Callout>
+
+Okay, let's get started installing your IBM MQ integration.
```
Comment (bradleycamacho) [inline src/install/ibm-mq/intro.mdx:34]:
```suggestion
To start set up the IBM MQ integration, follow these steps:
```

File: src/install/config/ibm-mq.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'windows'
+        - filePath: 'src/install/ibm-mq/linux/requirements-tarball.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'tarball'                
+  - filePath: 'src/install/ibm-mq/default-install.mdx'
```
Comment (bradleycamacho) [inline src/install/config/ibm-mq.yaml:40]: This doc is missing the step where the users pull down/download the actual integration. More info in comments below

File: src/install/ibm-mq/linux/install-linux.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+componentType: default
+headingText: Install the IBM MQ monitoring integration
+---
+
+1. Change directory to the integrations configuration folder by running:
```
Comment (bradleycamacho) [inline src/install/ibm-mq/linux/install-linux.mdx:6]: These looks good! The big issue here though, is these instructions refer to files that the user would not have at this point. The user needs to actually download the integration files reffered here. A bit hard to explain, but basically the users needs to "download" the integration by running a command. Once the command is run, the files used below appear.

See this step in the MongoDB doc: https://docs.newrelic.com/install/mongodb/#download-using-apt-4

File: src/install/config/ibm-mq.yaml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+              options: 
+                - value: 'windows'
+        - filePath: 'src/install/ibm-mq/linux/install-tarball.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'tarball'       
+  - filePath: 'src/install/ibm-mq/find-data.mdx'
+  - filePath: 'src/install/ibm-mq/config-sample-files.mdx'
+  - filePath: 'src/install/ibm-mq/configure.mdx'
```
Comment (bradleycamacho) [inline src/install/config/ibm-mq.yaml:59]: I'd pull the configure examples and metric refrence out of the actual steps. These are reference material, and explicitly a seperate concept from the procedural content provided in the steps. 

See here: https://docs.newrelic.com/install/mongodb/#instance-settings

File: src/install/ibm-mq/linux/requirements-linux.mdx [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+componentType: default
+headingText: Check the compatibility and requirements
+---
+
+* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic), minimum version 1.24.0.
```
Comment (bradleycamacho) [inline src/install/ibm-mq/linux/requirements-linux.mdx:6]: I'd pull this out into it's own step: https://docs.newrelic.com/install/mongodb/#install-the-infrastructure-agent-3

File: src/install/ibm-mq/linux/requirements-linux.mdx [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+* Linux distribution compatible with the [infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
```
Comment (bradleycamacho) [inline src/install/ibm-mq/linux/requirements-linux.mdx:7]:
```suggestion

```

Comment (bradleycamacho) [reply src/install/ibm-mq/linux/requirements-linux.mdx:7]: They can find this info when installing the infra agent

File: src/install/ibm-mq/linux/requirements-linux.mdx [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+* [IBM MQ Redistributable Client libraries](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqdev/redist/) on the agent box.
+* Check or add the libraries path in the `LD_LIBRARY_PATH` environment variable in Linux.
```
Comment (bradleycamacho) [inline src/install/ibm-mq/linux/requirements-linux.mdx:9]: Should this be it's own step?

**Review by bradleycamacho — CHANGES_REQUESTED** [review]
Looks really good for the most part! I had a few larger structural changes I left comments about. Feel free to start revision before our call, or just wait for our call so I can explain further!

**Review by bradleycamacho — CHANGES_REQUESTED** [review]
There's something weird going on with this entire PR. Looks like there are a few hundred files changed somehow!

I think you somehow pulled an old version of the repo onto your branch, and GH thought those "old files" were files you updated.

Basically, this PR is reverting most of the site to an old release. I'd try to delete those changes, or just close this PR and open a new one with just the IBM MQ work.

**Comment by nbaenam** [comment]
Hey @bradleycamacho, you're right. I'm going to close this PR and open a new one.

**Pull request closed without merging**